static/app/views/projectDetail/projectCharts.tsx

```diff
@@ -291,7 +292,7 @@ class ProjectCharts extends Component<Props, State> {
                 <ProjectBaseEventsChart
                   title={t('Apdex')}
                   help={getTermHelp(organization, apdexPerformanceTerm)}
-                  query="event.type:transaction"
+                  query={['event.type:transaction', query].join(' ').trim()}
```

Comment:
Should these use QueryResult now that they are a bit more complex?
https://github.com/getsentry/sentry/blob/69a803e3ead6a5d2ab1a06dfa2f27195cde8c43a/static/app/views/releases/detail/utils.tsx#L120-L124